src/azure-cli-core/azure/cli/core/commands/__init__.py

```diff
@@ -713,7 +712,7 @@ def _run_job(self, expanded_arg, cmd_copy):
         except Exception as ex:  # pylint: disable=broad-except
             if cmd_copy.exception_handler:
                 return cmd_copy.exception_handler(ex)
-            six.reraise(*sys.exc_info())
+            raise
```

Comment:
`sys.exc_info` can still be captured in exception body?

Reply:
https://docs.python.org/3/reference/simple_stmts.html#the-raise-statement

> If no expressions are present, `raise` re-raises the last exception that was active in the current scope.

Reply:
A stackoverflow post answering the difference of `raise` between Python 3 and Python 2: https://stackoverflow.com/questions/18188563/how-to-re-raise-an-exception-in-nested-try-except-blocks